internal/plan/plan_test.go

```diff
@@ -478,3 +479,38 @@ func (s *S) TestReadDirDupNames(c *C) {
 		}
 	}
 }
+
+func (s *S) TestMarshalLayer(c *C) {
+	layerStr := `
+summary: Simple layer
+description: A simple layer.
+services:
+    srv1:
+        summary: Service summary
+        default: start
+        override: replace
+        command: cmd arg1 "arg2 arg3"
+        after:
+            - srv2
+        before:
+            - srv3
+        requires:
+            - srv2
+            - srv3
+        environment:
+            - var1: val1
+            - var0: val0
+            - var2: val2
+    srv2:
+        override: replace
+        command: srv2cmd
+    srv3:
+        override: replace
+        command: srv3cmd
+`[1:]
```

Comment:
Can we please use the same function we use in the setup (now plan) package tests to indent this YAML properly and not need the [1:] at the end?  It's much nicer to read the code when the strings are aligned properly with the code instead of de-dented out of its context.

Reply:
Fixed.